### ISSUE TITLE
Is it possible to add more transactions into a bundle?

### ISSUE THREAD

**Issue by artiship** [weeaa/jito-go#8]
>   // max per bundle is 5 transactions
  txns := make([]*solana.Transaction, 0, 5)

**Comment by weeaa** [comment]
No, this is decided by Jito. However you are free to pass as many instructions as you want in a tx.
